### DISCUSSION TITLE
Support badges and badge styling

### DISCUSSION THREAD

**Discussion by el2iot2** [spectreconsole/spectre.console#458] [Ideas]
https://twitter.com/ThePracticalDev/status/1297551421903077377?s=20

Saw this npm package and thought it's concept would be a great addition to this library. Is it doable? Appropriate? In scope?

**Comment by patriksvensson** [comment]
@el2iot2 It would absolutely be doable and I have nothing against adding a "widget" that does this to the library.

![image](https://user-images.githubusercontent.com/357872/91019256-027f5600-e5f1-11ea-896d-9651cd7038f7.png)

![image](https://user-images.githubusercontent.com/357872/91019200-e7ace180-e5f0-11ea-9079-1c22d4f507b8.png)

**Comment by NikiforovAll** [comment]
This sounds fun. Do I understand correctly that this already could be done by using existing building blocks? We could make something more specific to draw badges. Could you please guide me, what is a good place to tackle this issue?
